signer/src/storage/model.rs

```diff
@@ -71,6 +72,13 @@ pub struct DepositRequest {
     #[sqlx(try_from = "i64")]
     #[cfg_attr(feature = "testing", dummy(faker = "100..100_000"))]
     pub max_fee: u64,
+    /// The relative lock time in the reclaim script.
+    #[sqlx(try_from = "i64")]
+    #[cfg_attr(feature = "testing", dummy(faker = "3..u32::MAX as u64"))]
+    pub lock_time: u64,
```

Comment:
So I understand, this has the same dual meaning as you specified in #380 around the `512` second per block interpretation yeah?

Reply:
Yup